.github/workflows/root-ci.yml

```
@@ -75,6 +75,10 @@ env:
   OS_INTERFACE: 'public'
   OS_REGION_NAME: 'cern'
 
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number }}
+  cancel-in-progress: ${{ github.event_name == 'pull_request' }}
```

```suggestion
  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.run_id }} 
  cancel-in-progress: ${{ github.event_name == 'pull_request' }}
```

---

@hahnjo, like this?

---

```suggestion
  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.run_id }}
  cancel-in-progress: true
```
should be sufficient.